### PR TITLE
UIEUS-527: Move delete button for an aggregator in the settings

### PR DESCRIPTION
**Pull request by elsenhans**
https://folio-org.atlassian.net/browse/UIEUS-527

### Description

As a user, I would like to be able to delete aggregator records in the settings directly from the detail view using an Actions menu, so that I don't have to switch to edit mode first. When I click the button, I want a pop-up window to appear warning me that I am about to delete the record.

**Use case**
Users must first switch to edit mode in order to delete the record. The Cross App SIG recommends placing the button in the record's detail view under the Actions menu, as this is the preferred location.

**Concept**
**Old position:** _Settings > Aggregators > Edit mode > top right corner_
The button must be removed.

**New position:** _Settings > Aggregators > Detail view_
Instead of the “Edit” button, an action menu is now used.
The Actions menu contains the following entries: Edit, Delete, Duplicate

**Acceptance criterias (UAT)**
- The delete button must be removed in edit mode.
- Detail view: The “Edit” button is no longer there.
- Detail view: There is a new “Actions” menu in the upper-right corner.
- Actions menu: The Actions menu contains the following entries Edit and Delete. The behavior of the entries when selected matches the description in the user story.
- The permissions must work.

### Approach

- Remove LastMenu, Confirmation Modal, delete function, delete state and delete button in `AggregatorForm.js`
- Enable Action menu by adding property `enableDetailsActionMenu` of `EntityManager`:
> If present and true, replaces the Edit button at top right of full records with an action menu offering Duplicate, Edit and Delete.
- Remove test for delete aggregator in `AggregatorForm.test.js`
- Add test for Action menu with available entries "Edit", "Duplicate" and "Delete".
-> Since the original purpose of this task was to make the delete button consistent with those in the other apps, no custom confirmation modal will be implemented (red delete button, custom modal message) here, knowing full well that it differs from the one we are using here https://github.com/folio-org/ui-erm-usage/pull/634

### CODE REVIEW

File: src/settings/Aggregators/AggregatorForm.js [hunk 1]
```diff
@@ -355,12 +293,7 @@ AggregatorForm.propTypes = {
   intl: PropTypes.object,
   invalid: PropTypes.bool,
   onCancel: PropTypes.func,
-  onRemove: PropTypes.func,
   pristine: PropTypes.bool,
-  stripes: PropTypes.shape({
```
Comment (slaemmer) [inline src/settings/Aggregators/AggregatorForm.js:360]: Stripes is still used in this file, or? Seems to me that only connect became obsolete. Therefore should retain hasPerm.

File: src/settings/Aggregators/AggregatorForm.test.js
Comment (slaemmer) [inline src/settings/Aggregators/AggregatorForm.test.js]: onRemove is removed in AggregatorForm - should be removed in the test as well.

File: src/settings/Aggregators/AggregatorForm.js [hunk 2]
```diff
@@ -328,19 +279,6 @@ const AggregatorForm = ({
                 </Row>
               </Accordion>
             </AccordionSet>
-
-            <ConfirmationModal
-              heading={<FormattedMessage id="ui-erm-usage.aggregator.form.delete.confirm.title" />}
```
Comment (slaemmer) [inline src/settings/Aggregators/AggregatorForm.js:333]: Removing this line results in an orphaned translation key - please remove it in en.json

File: src/settings/Aggregators/AggregatorForm.js (hunk 1 above)
Comment (elsenhans) [reply src/settings/Aggregators/AggregatorForm.js:360]: Thank you @slaemmer !
Restored `stripes` and `hasPerm` prop.

**Review by slaemmer — APPROVED** [review]
